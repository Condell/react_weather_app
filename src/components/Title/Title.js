import styled from "styled-components";

const Title = styled.h1`
  font-family: 'Abel', sans-serif;
  color: white;
`;

export default Title;