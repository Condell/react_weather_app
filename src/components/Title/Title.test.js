import React from "react";
import { shallow } from "enzyme";
import 'jest-styled-components'
import toJson from "enzyme-to-json"

import Title from "./Title";


describe("Title", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Title />);
  });

  it("should render", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});