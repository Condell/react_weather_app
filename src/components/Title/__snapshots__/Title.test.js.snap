// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render 1`] = `
.fUbstB {
  font-family: 'Abel', sans-serif;
  color: black;
  margin-top: 0px;
  margin-bottom: .5em;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
}

<h1
  className="sc-bdVaJa fUbstB"
/>
`;
